docs/process.md

```diff
@@ -129,7 +129,7 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
      the relevant issue name and include the issue number as well.<br>
      e.g. ` Incorrect error message when adding an existing instructor #1760`<br>
      In the PR description, mention the issue number in this format: `Fixes #1760`. 
-     Doing so will create an automatic reference from the issue to the pull request.<br>
+     Doing so will [automatically close](https://github.com/blog/1506-closing-issues-via-pull-requests) the related issue once the pull request is merged.<br>
```

Comment:
Good work! Just a minor suggestion, you can add the hyperlink to the entire phrase from "automatically ... merged" instead of stopping at "close".